Chapter06/MyERC20Token/contracts/MyERC20Token.sol

```diff
@@ -33,8 +33,8 @@ contract owned {
         admins[account] = true;    
     }
     function removeAdmin(address account) onlyOwner public {
-        require(account != address(0) && !admins[account]);
-        admins[account] = true;    
+        require(account != address(0) && admins[account]);
```

Comment:
I'm glad someone else caught this. I thought I was going crazy at first. Issue is there in the hard copy of the book on page 202 as well.